tests/integration/llm/client.py

```diff
@@ -686,7 +686,12 @@ def get_model_name():
         "batch_size": [1, 8],
         "seq_length": [256],
         "tokenizer": "google/flan-t5-xl",
-    }
+    },
+    "llama-3-1-8b": {
+        "batch_size": [1, 8],
+        "seq_length": [256],
+        "tokenizer": "NousResearch/Meta-Llama-3-8B",
```

Comment:
Should we use NousResearch/Meta-Llama-3.1-8B? Or both are same?

Reply:
yea that's a good catch - will update

Reply:
Okay, approved it for now. So you could update it and merge